src/components/form/form-button/form-button.component.js

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import I18n from 'i18n-js';
+import PropTypes from 'prop-types';
+import tagComponent from '../../../utils/helpers/tags/tags';
+import Button from '../../button';
+
+const text = (props) => {
+  const { formButtonName } = props;
+  const defaultValue = formButtonName.charAt(0).toUpperCase() + formButtonName.slice(1);
```

Comment:
I think Lodash's [`startCase()`](https://lodash.com/docs#startCase) function could be used here:

```js
import { startCase } from 'lodash';

const defaultValue = startCase(formButtonName);
```

Reply:
I would normally use that but we are trying to limit the use lodash with the idea of removing it as a dependency altogether at some point

Reply:
Understood :thumbsup: Maybe at some point we could create our own `toTitleCase()` helper function in `src/utils/helpers/text/text.js` ? Something like [this](https://stackoverflow.com/a/196991) maybe?
```js
function toTitleCase(str) {
  return str.replace(
    /\w\S*/g,
    function (txt) {
      return txt.charAt(0).toUpperCase() + txt.substr(1).toLowerCase();
    }
  );
}
```

Reply:
Yeah I like that idea, I'll add it in this PR

Reply:
Many thanks for creating `Text.titleCase()` :white_check_mark: